src/adapters/abstractAdapter.js

```diff
@@ -25,12 +25,34 @@ class Adapter {
 
   }
 
-  onTransfer (sourceAccount, targetAccount, amount) {
-    // Hook function
+  async onTransfer (sourceAccount, targetAccount, amount) {
+      this.sendTransferConfirmation(sourceAccount, amount)
```

Comment:
What's the idea behind this? onTransfer and sendTransferConfirmation do exactly the same thing (whereas onTransfer is more generic).

Reply:
My thinking is that 100% of all adapters should be sending confirmations; as such, it is appropriate to create a specific overridable function to handle this behavior. It's much clearer code for anyone who's wanting to create their own integration / adapter. 

Likewise, it's unclear if all or any integrations are going to want to perform additional functionality "onTransfer". Thus, I believe it makes sense to leave this is as being capable of being overridden, but not making it necessary. 

In short, my thinking is that if most "onTransfer" functions actually just send a transfer confirmation, it makes sense for the function to be called "sendTransferConfirmation" :)